src/components/common/modal/ModalActionsByFeedType.js

```diff
@@ -58,24 +58,32 @@ const ModalActionsByFeedType = ({ theme, styles, item, handleModalClose, navigat
   }, [showErrorDialog, setState, state, handleModalClose])
 
   const getPaymentLink = useMemo(() => {
-    const url = generateShareLink('send', {
-      paymentCode: item.data.withdrawCode,
-      reason: item.data.message,
-    })
+    try {
+      let result
+      const url = generateShareLink('send', {
+        paymentCode: item.data.withdrawCode,
+        reason: item.data.message,
+      })
 
-    let result
-
-    if (canShare) {
-      result = generateSendShareObject(url, item.data.amount, item.data.endpoint.fullName, currentUserName)
-    } else {
-      result = {
-        url: generateSendShareText(url, item.data.amount, item.data.endpoint.fullName, currentUserName),
+      if (canShare) {
+        result = generateSendShareObject(url, item.data.amount, item.data.endpoint.fullName, currentUserName)
+      } else {
+        result = {
+          url: generateSendShareText(url, item.data.amount, item.data.endpoint.fullName, currentUserName),
+        }
       }
-    }
 
-    fireEventAnalytics('Sharelink')
-
-    return result
+      fireEventAnalytics('Sharelink')
+      return result
+    } catch (e) {
+      log.error('getPaymentLink Failed', e.message, {
+        generateShareLink,
```

Comment:
you are printing methods to the log?
please try to understand where data.endpoint.fullName is set, what possible reasons would endpoint have to be null?
is the method getPaymentLink called for all cards?

Reply:
I checked all possible views, they all have the name of the endpoint. Therefore, I wrapped everything in a try/catch, in order to guarantee which feed might cause this problem. I removed methods logging.

Reply:
when creating a transaction, the formatEvent method forms the card body, it has a _extractFullName method that returns the fullName of the transaction owner or "GoodDollar". In Other places, fullName is preinstalled

Reply:
@StanislavShevchenko is .endpoint and .endpoint.fullName is always exists ?
could some feed item created without endpoint and have endpoint set later, e.g. on transaction complete or other cases ?
could it be mutated on some cases ?

Reply:
@serdiukov-o-nordwhale The endpoint is formed when formatting the card and preparing it for display. In UserStorage.formatEvent